src/screens/Admin/MemberAdmin.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import { Actions } from "react-native-router-flux";
+import { connect } from "react-redux";
+import { NavBar, Button, ButtonLayout, FilterList } from "../../components/general";
+import { SafeAreaView, View, Text } from "react-native";
+import { changePrivilegeOfMembers } from "../../ducks";
+import { MemberPanel } from "../../utils/actions"
+
+class MemberAdmin extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	render() {
+		const {
+			screenBackground,
+			contentContainer,
+			textStyle
+		} = styles;
+
+		return (
+			<SafeAreaView style = { screenBackground }>
+				<NavBar title = "Members" back onBack = { () => Actions.pop() } />
+				<View style = { contentContainer }>
+					<Text style = { textStyle }>Paid Members</Text>
+					<ButtonLayout>
+						{ [true, false].map(value =>
+							<FilterList
+								type = "Multiple"
+								CustomForm = { this.createButton(value) }
+								data = { this.makeUserList(value) }
+								regexFunc = { (data) => { return `${data.firstName} ${data.lastName}` } }
+								selectBy = { (data) => { return data.id } }
+								itemJSX = { (data) => MemberPanel(data) }
+								onSelect = { (selectedUsers) =>
+									changePrivilegeOfMembers(
+										selectedUsers.map((user) => { return user.id }), "paidMember", value) }
+							/>
+						) }
+					</ButtonLayout>
+				</View>
+			</SafeAreaView>
+		);
+	}
+
+	makeUserList(value) {
+		const {
+			userList
```

Comment:
if the onlything in here is userList and it's only used once it's better to just call this.props.userList